hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaTargetTypeManagement.java

```diff
@@ -139,6 +144,29 @@ public void delete(final Long targetTypeId) {
         return convertPage(targetTypeRepository.findAll(spec, pageable), pageable);
     }
 
+    @Override
+    public Page<TargetType> findByTargetTypeFilter(Pageable pageable, TargetTypeFilter targetTypeFilter) {
```

Comment:
I don't think that we need the TargetTypeFilter at all, as only the filterString is used (+ the searchText has the same beavior as the filterString in the end). The TargetTypeFilter can be replaced by a simple String parameter